### PR TITLE
step3 Add more information about Step 3

### LINKED ISSUE #4: Add more detail to Step 3
`docs/tasks/step-3.md` needs some more information.

### PR DESCRIPTION
**Pull request by SamLowth**

### CODE REVIEW

File: docs/tasks/step-3.md [hunk 1]
```diff
@@ -2,20 +2,57 @@
 
 ## a) Add table component
 
-On the `/all-books` page list all of the books using a [vuetify data table](https://vuetifyjs.com/en/components/data-tables/basics/).
+In this section, we will be adding a list of all books to the `all-books.vue` layout, using a [vuetify data table](https://vuetifyjs.com/en/components/data-tables/basics/). Similar to the book card, component, we will add the component using kebab-case naming standard, by adding the following to the template:
```
Comment (dev-steverob) [inline docs/tasks/step-3.md:5]: ` all-books.vue` page?

**Review by dev-steverob — APPROVED** [review]
LGTM

File: docs/tasks/step-3.md (hunk 1 above)
Comment (SamLowth) [reply docs/tasks/step-3.md:5]: as in change the wording 'ayout' to 'page' ?

**Review by Francis-Gurr — APPROVED** [review]
LGTM, let me know when it is ready to merge

**Pull request merged**